commons-fileupload2-core/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.apache.commons</groupId>
     <artifactId>commons-fileupload2</artifactId>
-    <version>2.0.0-M2-SNAPSHOT</version>
+    <version>2.0.0-M3-SNAPSHOT</version>
```

Comment:
Don't change the version please.

Reply:
Despite the changes, you'd like to leave this at 2.0.0-M2 ? Or you want to change the version yourself?

Reply:
2.0.0-M2 will be the next release, 2.0.0-M2-SNAPSHOT is the temporary version until the release. Don't change any of these versions in the poms.

Reply:
I had to change version as artifacts were modified and that change needed to be reflected via version change / I needed to refer to those particular modified artifacts.. Those artifacts are already used in internal projects. 

Do you want me to change those versions for you just for this PR or do you mind changing those back to M2 when you merge it, as I see you as author of all commits @ https://github.com/apache/commons-fileupload/commits/master (at least first several pages) anyway ?